packages/component-library/components/Spacing/Base.module.scss

```diff
@@ -1,19 +1,19 @@
 @import "~@kaizen/component-library/styles/fonts";
-@import "~@kaizen/deprecated-component-library-helpers/styles/type"; // just for the grid unit @TODO - update when we move to dart-sass
+@import "~@kaizen/design-tokens/sass/spacing";
```

Comment:
I've just updated this to remove dependence on deprecated comp lib helpers

Reply:
It looks so weird to have them all reference `$spacing-md`, but I guess it's the same for now 🤷🏻 

I wonder if we'd still have this list when we have the new tokens 🤔

Reply:
hahaha yeah, we'll probably remove the need for this in its entirety 😆